src/test/java/com/rackspace/salus/telemetry/ambassador/config/GrpcConfigVaultTest.java

```diff
@@ -73,7 +73,8 @@
         GRpcAutoConfiguration.class
     },
     properties = {
-        "grpc.port=0"
+        "grpc.port=0",
+        "salus.ambassador.public-zone-tenants= {'aaaaaa'}"
```

Comment:
That doesn't seem to be a valid properties value, but maybe Spring Boot can parse that. I was thinking
```suggestion
        "salus.ambassador.public-zone-tenants=aaaaaa"
```